examples/02-plot/silhouette.py

```diff
@@ -0,0 +1,91 @@
+"""
+Silhouette Highlight
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
One too many wiggles, although to be honest I'm not sure how important that is for sphinx.
```suggestion
~~~~~~~~~~~~~~~~~~~~
```